### DISCUSSION TITLE
Azuracast WebDj issue

### DISCUSSION THREAD

**Discussion by Shahzaib-jpg** [AzuraCast/AzuraCast#6065] [Q&A]
I have installed azuracast on docker and ubuntu and running it on localhost. but azuracast webdj feature is not working very well. It does not giving me options about metadata.
![Screenshot 2023-02-07 110646](https://user-images.githubusercontent.com/124657191/217162068-1e3daafd-8b64-472f-8d03-d6587438cba3.png)
And it also giving me an error when I tries to start a live stream using webDj that

**Comment by BusterNeece** [comment]
Metadata options appear once streaming is started.
